README.md

```diff
@@ -1,34 +1,41 @@
+# Icons Legend
```

Comment:
I would like to use `##` for "section" headers, because the `#` is like `H1` in HTML - should be only one per page. In our case it should be the repository name. 
I am going to change the header level in the rest of headers, but please change only this one.

Reply:
Ok, we didn't talk about what do I do now :D I've received your comments, and should I now go the README.md file and implement my changes there? Or should I do it somewhere here, in the "files changed" tab?

Reply:
Hey hey, I also vote for `##` - nevermind my original formatting, it was just an MVP :)

Reply:
@kat-kap we can change the rest of headers in another PR, or you can do it here. It's up to you :)

Reply:
I am not sure i I understand you question, but to change the header you have to create a new commit on your feature branch (the one which you want to merge into main )

Reply:
I've updated only the "Icons" header, so go ahead and update the remaining ones :)